tests/baselines/reference/tsNoCheckForTypescript.js

```diff
@@ -0,0 +1,57 @@
+//// [file.ts]
+// @ts-nocheck
+
+export const a = 1 + {}; // This is an error, ofc, `Operator '+' cannot be applied to types '1' and '{}'`, which will be suppressed by the `nocheck` comment
+
+export interface Aleph {
+    q: number;
+}
+
+export class Bet implements Aleph {
+    q: string = "lol" // And so will this implements error
+}
+
+
+//// [file.js]
+"use strict";
+// @ts-nocheck
+exports.__esModule = true;
+exports.a = 1 + {}; // This is an error, ofc, `Operator '+' cannot be applied to types '1' and '{}'`, which will be suppressed by the `nocheck` comment
+var Bet = /** @class */ (function () {
+    function Bet() {
+        this.q = "lol"; // And so will this implements error
+    }
+    return Bet;
+}());
+exports.Bet = Bet;
+
+
+//// [file.d.ts]
+export declare const a: any;
+export interface Aleph {
+    q: number;
+}
+export declare class Bet implements Aleph {
+    q: string;
+}
+
+
+//// [DtsFileErrors]

```

Comment:
I just decided to have declaration emit on to capture the true "garbage in, garbage out" aspect of error suppression. Again, this was already doable with multiple individual `//@ts-ignore`s so....